.github/workflows/pypi-release.yml

```diff
@@ -8,23 +8,37 @@ jobs:
   build-artifacts:
     runs-on: ubuntu-latest
     if: github.repository == 'ARM-DOE/pyart'
+    defaults:
+      run:
+        shell: bash -l {0}
     steps:
       - uses: actions/checkout@v2
         with:
           fetch-depth: 0
-      - uses: actions/setup-python@v2
-        name: Install Python
+       # Install dependencies
+      - name: Setup Conda Environment
+        uses: conda-incubator/setup-miniconda@v2
         with:
-          python-version: 3.8
+          environment-file: continuous_integration/environment-ci.yml
+          activate-environment: pyart-dev
+          auto-activate-base: false
+          mamba-version: '*'
+          use-mamba: true
+          miniforge-variant: Mambaforge
+          extra-specs: python=3.8
+
+      - name: Set TRMM RSL path
+        run: |
+          export RSL_PATH=$CONDA/pkgs/trmm_rsl-1.49-3
```

Comment:
Add in a line here `conda install -c conda-forge trmm_rsl` instead of adding to the environment.yml for the CI